src/dgpost/utils/plot.py

```diff
@@ -0,0 +1,126 @@
+"""
+``plot``: Plots columns of a DataFrame.
+
+The function :func:`dgpost.utils.plot.plot` processes the below specification
+in order to generate a plot:
+
+.. code-block:: yaml
+
+    plot:
+      - table:      !!str       # internal DataFrame name
+        style:      !!dict      # optional dictionary to adjust style of plot, see :class:`matplotlib.rcParams`
+        fig_args:   !!dict      # optional kwargs for the generated :class:`matplotlib.figure.Figure`
+        nrows:      !!int       # optional int defining the number of rows of the grid for :class:`matplotlib.gridspec.GridSpec`
+        ncols:      !!int       # optional int defining the number of columns of the grid for :class:`matplotlib.gridspec.GridSpec`
+        save:                   # optional to save the generated plot, `fname` required and any additional kwargs gets passed to :func:`matplotlib.figure.Figure.savefig`
+          fname:    !!str       # path, where the file is saved
+        ax_args:                # sequence of arguments for the different axes
```

Comment:
Can you move `ax_args:` below `fig_args` in the docstring? The docs will be a little more structured then.

Reply:
Sure